src/south/modelsinspector.py

```diff
@@ -40,7 +40,9 @@ def convert_on_delete_handler(value):
         else:
             # This is totally dependent on the implementation of django.db.models.deletion.SET
             func_name = getattr(value, '__name__', None)
-            if func_name == 'set_on_delete':
+            if func_name == 'SET_NULL':
```

Comment:
This changes to be its own function that doesn't call `set_on_delete` anymore, so we just check for `SET_NULL` now